### PR TITLE
Added function to get worker flags in blueprints

### PR DESCRIPTION
**Pull request by Vatyx**
#### Description
Added blueprint callable function which can retrive a worker flag given a its name.

#### Tests
Tried it locally with different flags.

#### Documentation
Undocumented for now.

#### Primary reviewers
@joshuahuburn @m-samiec

### CODE REVIEW

File: SpatialGDK/Source/Private/Interop/SpatialDispatcher.cpp [hunk 1]
```diff
@@ -82,12 +83,14 @@ void USpatialDispatcher::ProcessOps(Worker_OpList* OpList)
...
 
 		case WORKER_OP_TYPE_FLAG_UPDATE:
+			USpatialWorkerFlags::ApplyWorkerFlagUpdate(Op->flag_update);
 			break;
 		case WORKER_OP_TYPE_LOG_MESSAGE:
 			UE_LOG(LogSpatialView, Log, TEXT("SpatialOS Worker Log: %s"), UTF8_TO_TCHAR(Op->log_message.message));
 			break;
 		case WORKER_OP_TYPE_METRICS:
 			break;
+
```
Comment (danielimprobable) [inline SpatialGDK/Source/Private/Interop/SpatialDispatcher.cpp]: Why the newline?

**Comment by girayimprobable** [comment]
Add a JIRA ticket please, and if you're not delivering documentation as part of this PR make sure to move it to the "documenting" status.

**Comment by danielimprobable** [comment]
I think docs should be part of this PR rather than merging it and leaving it in the swim lane

File: SpatialGDK/Source/Private/Interop/SpatialWorkerFlags.cpp [hunk 2]
```diff
@@ -0,0 +1,33 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+
+#include "SpatialWorkerFlags.h"
+#include <WorkerSDK/improbable/c_worker.h>
+
+TMap<FString, FString> USpatialWorkerFlags::WorkerFlags;
+
+bool USpatialWorkerFlags::GetWorkerFlag(const FString& Name, FString& Value)
```
Comment (m-samiec) [inline SpatialGDK/Source/Private/Interop/SpatialWorkerFlags.cpp]:
```suggestion
bool USpatialWorkerFlags::GetWorkerFlag(const FString& Name, FString& OutValue)
```

File: SpatialGDK/Source/Private/Interop/SpatialWorkerFlags.cpp [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+{
+	if (FString* ValuePtr = WorkerFlags.Find(Name))
+	{
+		Value = *ValuePtr;
```
Comment (m-samiec) [inline SpatialGDK/Source/Private/Interop/SpatialWorkerFlags.cpp]:
```suggestion
		OutValue = *ValuePtr;
```

File: SpatialGDK/Source/Public/Interop/SpatialWorkerFlags.h [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+{
+	GENERATED_BODY()
+
+	/** Gets value of a worker flag. Must be connected to SpatialOS to properly work.
+	 * @param Name - Name of worker flag
+	 * @param Value - Value of worker flag
+	 * @return - If worker flag was found.
+	 */
+	UFUNCTION(BlueprintCallable, Category="SpatialOS")
+	static bool GetWorkerFlag(const FString& Name, FString& Value);
```
Comment (m-samiec) [inline SpatialGDK/Source/Public/Interop/SpatialWorkerFlags.h]:
```suggestion
	static bool GetWorkerFlag(const FString& Name, FString& OutValue);
```

**Pull request merged**